includes/classes/config/wallets.php

```diff
@@ -42,20 +46,22 @@ protected function add($wallet) {
             return false;
         }
 
-        $class = 'Wallets_' . ucwords(strtolower($wallet['currency']));
+        $class = Wallets::getSupportedWalets();
+        $class = $class[$wallet['currency']];
 
         if (class_exists($class)) {
             $walletData = array();
             $addessData = array();
```

Comment:
I know this was here before, can someone refactor this (and few other instances) to say $addressData please. Thanks.